cmd/commands/wiring.go

```diff
@@ -189,7 +190,7 @@ func installKubeConfigSupport(command *cobra.Command, client *core.Client, kc *c
 	kubeConfigPath := ""
 	masterURL := ""
 
-	command.PersistentFlags().StringVar(&kubeConfigPath, "kubeconfig", "~/.kube/config", "the `path` of a kubeconfig")
+	command.PersistentFlags().StringVar(&kubeConfigPath, "kubeconfig", fmt.Sprintf("~%[1]c.kube%[1]cconfig", filepath.Separator), "the `path` of a kubeconfig")
```

Comment:
Actually, this change isn't needed because Windows accepts paths with forward slashes instead of backward slashes. However, when we parse Windows paths, we need to be prepared to accept backslashes which is why we can't parse for strings starting `~/`.